docs/admin/pricing.mdx

```diff
@@ -2,10 +2,40 @@
 
 Visit the [pricing page](https://about.sourcegraph.com/pricing) for up-to-date pricing. 
 
-## How are active users calculated?
+## How are active users calculated for **Sourcegraph Cody**?
 
-Any user who accesses Sourcegraph in a given month is considered a monthly active user (MAU). This includes but is not limited to:
+### Billable active users
 
+_Note: this only applies to Enterprise Cody contracts. Cody Pro users pay for a seat every month, regardless of usage._
+
+A billable active Cody user is one who is signed in to their Enterprise Sourcegraph account and actively
+interacts with Cody (e.g. they see suggested autocompletions, they run commands or chat with
+Cody, they start new discussions, clear chat history, or copy text from chats, they change Cody
+settings, and more). 
+
+Simply having Cody installed is not enough to be considered a billable user.
```

Comment:
Should we clarify this as `Cody installed and authenticated against the instance` to be extra clear here?

Reply:
Yep good call. I adjusted the wording slightly to `Cody installed and signed in`